itest/addrs_test.go

```diff
@@ -184,6 +190,243 @@ func testMultiAddress(t *harnessTest) {
+			GlobalSyncConfigs: globalConfigs,
+		},
+	)
+	require.NoError(t.t, err)
+
+	configResp, err := bob.UniverseClient.QueryFederationSyncConfig(
+		ctxb, &unirpc.QueryFederationSyncConfigRequest{},
+	)
+	require.NoError(t.t, err)
+
+	// Ensure that the global configs are set as expected.
+	require.Equal(t.t, len(configResp.GlobalSyncConfigs), 2)
+
+	for i := range configResp.GlobalSyncConfigs {
+		config := configResp.GlobalSyncConfigs[i]
+
+		// Match proof type.
+		switch config.ProofType {
+		case unirpc.ProofType_PROOF_TYPE_ISSUANCE:
+			require.False(t.t, config.AllowSyncInsert)
+			require.False(t.t, config.AllowSyncExport)
+
+		case unirpc.ProofType_PROOF_TYPE_TRANSFER:
+			require.False(t.t, config.AllowSyncInsert)
+			require.False(t.t, config.AllowSyncExport)
+
+		default:
+			t.Fatalf("unexpected global proof type: %s",
+				config.ProofType)
+		}
+	}
+
+	// Mint more assets with the main node, which should not sync to Bob.
+	secondRpcAssets := MintAssetsConfirmBatch(
+		t.t, miner, t.tapd, []*mintrpc.MintAssetRequest{
+			simpleAssets[1], issuableAssets[1],
+		},
+	)
+	require.Len(t.t, secondRpcAssets, 2)
+
+	// Verify that Bob will not sync to Alice by default by manually
+	// triggering a sync.
+	syncDiff, err := bob.SyncUniverse(ctxt, &unirpc.SyncRequest{
+		UniverseHost: t.tapd.rpcHost(),
+		SyncMode:     unirpc.UniverseSyncMode_SYNC_ISSUANCE_ONLY,
+	})
+	require.NoError(t.t, err)
+	require.Len(t.t, syncDiff.SyncedUniverses, 0)
+
+	// This helper restarts Bob, disables global universe sync, and adds
+	// Alice to his federation.
+	restartBobNoUniSync := func(disableSyncer bool) {
+		require.NoError(t.t, bob.stop(!*noDelete))
+		bob = setupTapdHarness(
+			t.t, t, t.lndHarness.Bob, nil,
+			func(params *tapdHarnessParams) {
+				params.addrAssetSyncerDisable = disableSyncer
+			},
+		)
+
+		_, err = bob.UniverseClient.SetFederationSyncConfig(
+			ctxb, &unirpc.SetFederationSyncConfigRequest{
+				GlobalSyncConfigs: globalConfigs,
+			},
+		)
+		require.NoError(t.t, err)
+
+		_, err = bob.AddFederationServer(
+			ctxt, &unirpc.AddFederationServerRequest{
+				Servers: []*unirpc.UniverseFederationServer{
+					{
+						Host: t.tapd.rpcHost(),
+					},
+				},
+			},
+		)
+		require.NoError(t.t, err)
+		AssertUniverseStats(t.t, bob, 0, 0, 0, 0)
+	}
+
+	// If we restart Bob with the syncer disabled and no automatic sync
+	// with Alice, he should be unable to make an address for the new
+	// assets.
+	restartBobNoUniSync(true)
+	firstAsset = secondRpcAssets[0]
+	_, err = bob.NewAddr(ctxt, &taprpc.NewAddrRequest{
+		AssetId:      firstAsset.AssetGenesis.AssetId,
+		Amt:          firstAsset.Amount,
+		AssetVersion: firstAsset.Version,
+	})
+	require.ErrorContains(t.t, err, "asset group is unknown")
+
+	// Restart Bob again with the syncer enabled. Bob should be able to make
+	// an address for both new assets minted by Alice, even though he has
+	// not synced their issuance proofs.
+	restartBobNoUniSync(false)
```

Comment:
:+1: 
Last note w.r.t being able to enable/disable JIT sync: Is the "global sync" (fetch all) and JIT sync going to live side-by-side in the future? Feels to me that if JIT is disabled then global sync must be present, and if JIT is enabled global has to be off, as these two sort of overlap

Reply:
does that lead us to 2 future modes for sync?

Reply:
Good Q: So if global sync is on, the syncer setting sort of doesn't matter; the only way to not know about an asset would be if it was pushed to a federation member after your last sync (up to 10 mins ago).

The default with this PR and the others in main is "JIT sync on, global sync off". With manual global sync still possible.

With JIT on anything you sync from a global sync, you haven't interacted with yet, so I think that's a reasonable default.